plenum/common/ledger_uncommitted_tracker.py

```diff
@@ -0,0 +1,40 @@
+from stp_core.common.log import getlogger
+from collections import deque
+from common.exceptions import PlenumValueError, LogicError
+
+logger = getlogger()
+
+
+class LedgerUncommittedTracker:
+
+    def __init__(self):
+        self.un_committed = deque()
+
+    def apply_batch(self, state_root, ledger_size):
+
+        un_committed_state = ()
+
+        if state_root != "":
```

Comment:
Is it expected that we use "" and not, for example, None to detect that no state root is passed?
If we want a real validation of the state root here, then we may use `MerkleRootField`.
But actually I think that for such a simple class this validation may be a bit too complex and time consuming.
I would say that it's enough to check for None only here, so that
```
# validation
if state_root is None:
   raise PlenumValueError
if ledger_size is None or ledger_size <= 0:
   raise PlenumValueError

self.un_committed.append(state_root, ledger_size ))

```

Reply:
I agree with this. I will make these changes